clang/test/CodeGenSYCL/int_header1.cpp

```diff
@@ -1,6 +1,12 @@
 // RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -sycl-std=2020 -fsycl-int-header=%t.h %s -o %t.out
 // RUN: FileCheck -input-file=%t.h %s
 
+// Check if forward declarations of kernel names in anonymous namespace are in
```

Comment:
Can you change the 2nd and 3rd to 'CHECK-NEXT' to make sure here?   Also, are we putting comments on our close braces?  if so, I'd like those added to the check line as well.

Reply:
> Also, are we putting comments on our close braces? if so, I'd like those added to the check line as well.
There aren't any comments on the close braces in the integration header generated.